frontend/src/app/Settings/Users/ManageUsersContainer.tsx

```diff
@@ -43,22 +43,52 @@ interface UserData {
   users: SettingsUser[];
 }
 
-// const GET_FACILITIES = gql`
-//   query GetFacilities {
-//     organization {
-//       testingFacility {
-//         id
-//         name
-//       }
-//     }
-//   }
-// `;
-
-// interface FacilityData {
-//   organization: {
-//     testingFacility: UserFacilitySetting[];
-//   };
-// }
+const UPDATE_USER_ROLE = gql`
+  mutation UpdateUserRole($id: ID!, $role: OrganizationRole!) {
+    updateUserRole(id: $id, role: $role)
+  }
+`;
+
+const DELETE_USER = gql`
+  mutation SetUserIsDeleted($id: ID!, $deleted: Boolean!) {
+    setUserIsDeleted(id: $id, deleted: $deleted) {
+      id
+    }
+  }
+`;
+
+const ADD_USER_TO_ORG = gql`
+  mutation AddUserToCurrentOrg(
+    $firstName: String
+    $lastName: String!
+    $email: String!
+  ) {
+    addUserToCurrentOrg(
+      firstName: $firstName
+      lastName: $lastName
+      email: $email
+    ) {
+      id
+    }
+  }
+`;
+
+const GET_FACILITIES = gql`
+  query GetFacilities {
```

Comment:
This should have a unique query name to make debugging easier (at least Tim tells me). There's a ticket to do that in other files but we might as well fix the ones we touch. Maybe `GetFacilitiesForManageUsers` if there isn't a better name.

Reply:
This seems like a suitable name, making the change